tests/test_plugin.py

```diff
@@ -214,5 +214,55 @@ def test_it_adapts_splits_based_on_new_and_deleted_tests(
         ]
 
 
+class TestHasExpectedOutput:
+    def test_prints_splitting_summary(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", "1", "--group", 1)
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Running group 1/1 (10/10) tests" in outerr.out
+
+    def test_prints_splitting_summary_when_group_missing(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", "1")
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because the `group` argument is missing" in outerr.out
+
+    def test_prints_splitting_summary_when_splits_missing(self, example_suite, capsys):
+        result = example_suite.inline_run("--group", "1")
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because the `splits` argument is missing" in outerr.out
+
+    def test_prints_splitting_summary_when_splits_missing(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", 1, "--group", "1")
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because the durations_report is missing" in outerr.out
+
+    def test_prints_splitting_summary_when_durations_missing(self, example_suite, capsys, durations_path):
+        test_name = "test_prints_splitting_summary_when_durations_missing"
+        with open(durations_path, "w") as f:
+            json.dump([[f"{test_name}0/{test_name}.py::test_1", 0.5]], f)
+
+        result = example_suite.inline_run(
+            "--splits", 1, "--group", "1", "--durations-path", durations_path, "--store-durations"
+        )
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because we are storing durations" in outerr.out
```

Comment:
I guess this and the one above ("Not splitting tests because the durations_report is missing") will collide with the stuff discussed in https://github.com/jerry-git/pytest-split/issues/11

Reply:
Yes, but I think it's fine to keep this as is and change it again later. What do you think?

Reply:
I'm afraid there will be quite big conflicts with this one https://github.com/jerry-git/pytest-split/pull/14

Reply:
Yes, I've added a comment there. I think it would be good to split the PR that "updates printing" from one that updates storing of durations.

Reply:
I'm happy to resolve the conflicts in #14. I guess this message would no longer be relevant but that's OK 🙂